### PR TITLE
Improve error wording

### PR DESCRIPTION
**Pull request by evrardjp**
## Why is this PR needed?

Some of the errors thrown by skuba are not really explicit for
our customers.

This should fix it.

## What does this PR do?

Clarify error messages

## Anything else a reviewer needs to know?

Nothing

## Info for QA

Nothing new to test, as the behaviour hasn't changed.
However, if any automated tested relies on strings (they
shouldn't!), those might need adapting.

### Related info

See also for the reason of existence of this PR, and its wording
used.
* https://github.com/SUSE/skuba/pull/1354/files
* https://github.com/SUSE/skuba/pull/1353/files

### Status **BEFORE** applying the patch

Broken English

### Status **AFTER** applying the patch

Less broken English

## Docs

Nothing to change in the docs, AFAIK.

# Merge restrictions

(Please do not edit this)

We are in *v4-maintenance phase*, so we will restrict what can be merged to prevent unexpected surprises:

    What can be merged (merge criteria):
        2 approvals:
            1 developer: code is fine
            1 QA: QA is fine
        there is a PR for updating documentation (or a statement that this is not needed)

### CODE REVIEW

**Review by chentex — DISMISSED** [review]
LGTM

**Review by mssola — DISMISSED** [review]

File: internal/pkg/skuba/upgrade/node/versions.go [hunk 1]
```diff
@@ -98,14 +98,14 @@ func (nviu NodeVersionInfoUpdate) IsFirstControlPlaneNodeToBeUpgraded(client cli
...
-		errorMessages = append(errorMessages, fmt.Sprintf("Cannot use this version of skuba to upgrade to kubernetes version %s. Please use a skuba version below 2.0.0 instead.", skubaconstants.LastCaaSP4KubernetesVersion))
+		errorMessages = append(errorMessages, fmt.Sprintf("Cannot use this version of skuba to upgrade to kubernetes version %s. Please use a skuba from CaaSP 4 on SLES 15 SP1 instead (version below 2.0.0).", skubaconstants.LastCaaSP4KubernetesVersion))
 	}
 	isKubeletTooOld, err := nviu.Update.KubeletVersion.Compare("1.17.0")
 	if err != nil {
 		return err
 	}
 	if isKubeletTooOld < 0 {
-		errorMessages = append(errorMessages, fmt.Sprintf("Make sure your node is at least to version %s before upgrading", skubaconstants.LastCaaSP4KubernetesVersion))
+		errorMessages = append(errorMessages, fmt.Sprintf("Make sure your node is at least to version %s before trying an upgrade with this version of skuba for CaaSP 4.5 on SLES 15 SP2", skubaconstants.LastCaaSP4KubernetesVersion))
```
Comment (kkaempf) [inline internal/pkg/skuba/upgrade/node/versions.go:108]: should be "...is at least _at_ version %s ..."

Comment (chentex) [reply internal/pkg/skuba/upgrade/node/versions.go]: Thank you, changed the file.

**Review by chentex — APPROVED** [review]
LGTM

**Review by mssola — APPROVED** [review]

**Pull request merged**